### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by ssaviak**
[DEMO LINK](https://ssaviak.github.io/layout_landing-page/)

### CODE REVIEW

**Comment by ssaviak** [comment]
I forgot to redeploy, pls check it again

**Review by olya-shyian — CHANGES_REQUESTED** [review]
Good job! Let's do some fixes

1. Change text color on hover for phone, email, and address
image

2. Add cursor pointer to submit button as well
image

3. Make this hover effect smoothly
image

4. When you click in the menu, for example on Products, then you should be dynamically transferred to "Recommended" section etc.

File: src/styles/about-us.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    @include onTablet {
+      margin-top: 0;
+    }
+  }
+
+  &__description {
+    margin-top: 24px;
+
+    color: $main-color;
+    font-family: "Space Mono", Arial, Helvetica, sans-serif;
```
Comment (DanilWeda) [inline src/styles/about-us.scss:53]: Move to variables

File: src/styles/category.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
+.category {
+  font-family: 'Space Grotesk', sans-serif;
```
Comment (DanilWeda) [inline src/styles/category.scss:2]: use variable

File: src/styles/contact-us.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    margin-top: 48px;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+  }
+
+  &__input {
+    height: 48px;
+    padding-inline: 16px;
+    color: #7E7E83;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:22]: move to variables, write in lowercase

File: src/styles/contact-us.scss [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+    font-family: 'Space Mono',sans-serif;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:24]: use variable

File: src/styles/contact-us.scss [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    font-size: 14px;
+    font-weight: 400;
+
+    border-radius: 8px;
+    background-color: #F8F8FA;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:29]: move to variables, write in lowercase

File: src/styles/contact-us.scss [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    border:none;
+
+    &::placeholder {
+      left: 16px;
+      color: $input-font-default-color;
+
+      font-family: 'Space Mono', sans-serif;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:36]: use variable

File: src/styles/contact-us.scss [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+        font-size: 14px;
+      }
+    }
+  }
+
+  &__text-area {
+    padding: 14px 16px;
+
+    color: $input-font-default-color;
+    font-family: 'Space Mono', sans-serif;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:62]: use variable 
You can write this font for the parent element, and then - inherit it for child elements.

File: src/styles/contact-us.scss [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 140%;
+
+    border-radius: 8px;
+    background-color: #F8F8FA;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:68]: move to variables, write in lowercase

File: src/styles/contact-us.scss [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+      color: $main-color;
+      &::placeholder {
+        color: $main-color;
+      }
+    }
+  }
+
+  &__send {
+    border-radius: 8px;
+    border: 2px solid #F0F1F4;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:87]: move to variables, write in lowercase

File: src/styles/contact-us.scss [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+    background: #F8F8FA;
+    width: 100%;
+    height: 56px;
+
+    font-family: 'Space Grotesk', sans-serif;
```
Comment (DanilWeda) [inline src/styles/contact-us.scss:92]: use variable

File: src/styles/contacts.scss [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+    margin-top: 0;
+  }
+}
+
+.contacts {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+
+  font-family: 'Space Mono',Arial, Helvetica, sans-serif;
```
Comment (DanilWeda) [inline src/styles/contacts.scss:17]: use variable

File: src/styles/contacts.scss [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 140%;
+
+  color: #191919;
```
Comment (DanilWeda) [inline src/styles/contacts.scss:22]: move to variables

File: src/styles/contacts.scss [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+  &__label {
+    color: #9393A3;
```
Comment (DanilWeda) [inline src/styles/contacts.scss:25]: move to variables, write in lowercase

File: src/styles/contacts.scss [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+    font-family: 'Space Grotesk', sans-serif;
```
Comment (DanilWeda) [inline src/styles/contacts.scss:26]: use variable

File: src/styles/utils/variables.scss [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+$item-scale-size: 1.03;
+$item-scale-duration: .5s;
+$transition-duration: .3s;
+$appear-duration: 2s;
+$delay-header: .3s;
+$delay-header-title: .6s;
+$delay-recommend: .9s;
+$delay-categories: 1.2s;
+$delay-contact-us: 1.5s;
+$delay-about-contacts-us: 1.8s;
```
Comment (DanilWeda) [inline src/styles/utils/variables.scss:18]: Use lowercase for all variables

File: src/styles/product-card.scss [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 140%;
+  }
+
+  &__description {
+    margin-top: 8px;
+
+    font-family: 'Space Mono', sans-serif;
```
Comment (DanilWeda) [inline src/styles/product-card.scss:26]: use variable

File: src/styles/product-card.scss [hunk 17]
```diff
@@ -0,0 +1,62 @@
+.product-card {
+  color: $main-color;
+  font-family: 'Space Grotesk', sans-serif;
```
Comment (DanilWeda) [inline src/styles/product-card.scss:3]: use variable

File: src/styles/menu.scss [hunk 18]
```diff
@@ -0,0 +1,110 @@
...
+  }
+
+  @include onDesktop {
+    &__item:hover {
+      display: block;
+      position: relative;
+      &::after {
+        content: '';
+        height: 1px;
+        background-color: #292929;;
```
Comment (DanilWeda) [inline src/styles/menu.scss:45]: move to variables

**Review by DanilWeda — CHANGES_REQUESTED** [review]
Great job. 
Check code-style comments. 
Also, pay attention to menu nav elements ( transition), textarea resize, and centering header of your site with a header on the menu ( when open).

**Review by olya-shyian — APPROVED** [review]
Amazing work! 🤩
I will approve it but could you make these last changes?

1. You made a hover effect on the 'contacts', it’s good, but there the color change is not noticeable at all. can you please use a slightly different color... for example, gray

2. You added an anchor on 'products' and the page will scroll - it's a good) now  please also add another anchor on 'store', which will send you to the next section "Browse Nothing products by category"

image
image

**Pull request closed without merging**